modules/openapi-generator/src/main/resources/Java/typeInfoAnnotation.mustache

```diff
@@ -1,6 +1,7 @@
 {{#jackson}}
 
-@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, include = JsonTypeInfo.As.EXISTING_PROPERTY, property = "{{{discriminator.propertyBaseName}}}", visible = true)
+@com.fasterxml.jackson.annotation.JsonIgnoreProperties(value = "{{{discriminator.propertyBaseName}}}", allowSetters = true)
```

Comment:
Can you add a code comment here so that future users of the generator can understand _why_ you've configured `JsonIgnoreProperties` ?

Maybe something like...

```
// Configure JsonIgnoreProperties so that the type property is generated by Jackson during serialization.
// allowSetters allows the type to be set during deserialization.
```

Reply:
6632b6e907780003de9eca1d3be4424b691e7e5a